frontend/src/views/FarmInformation/FarmInformation.tsx

```diff
@@ -39,8 +51,22 @@ export default function FarmInformation() {
   const [rawAnimalNames, setRawAnimalNames] = useState<{ [id: string]: string }>({});
 
   // Props for region selections
-  const [regionOptions, setRegionOptions] = useState<{ value: number; label: string }[]>([]);
-  const [subregionOptions, setSubregionOptions] = useState<{ value: number; label: string }[]>([]);
+  const [regionOptions, setRegionOptions] = useState<Array<SelectOption>>([]);
+  const [subregionOptions, setSubregionOptions] = useState<Array<SelectOption>>([]);
+
+  // Initialize year list
+  const [yearOptions, setYearOptions] = useState<Array<SelectOption>>([]);
+
+  useEffect(() => {
+    const currentYear = new Date().getFullYear();
+    const yearArray = [];
+
+    for (let i = currentYear; i > currentYear - 20; i -= 1) {
+      yearArray.push({ id: i.toString(), label: i.toString() });
```

Comment:
Were we asked to change the Year field to a dropdown?

Reply:
I recall hearing something along the lines, but after checking the design and ticket I don't see it. I'm going to confirm with Jordan on the UI.

Reply:
Confirmed, he's okay with it.